### PR TITLE
Add missing elements of SQA templates for misc and phase_field modules

### PR DESCRIPTION
**Pull request by bwspenc**
ref #15968

### CODE REVIEW

**Comment by cticenhour** [comment]
Documentation URL from March 9th is broken - running CI again to refresh status.

File: modules/misc/doc/content/modules/misc/sqa/misc_srs.md [hunk 1]
```diff
@@ -1 +1,26 @@
 !template load file=sqa/module_srs.md.template category=misc module=Misc
+
+!template! item key=system-scope
+!! system-scope-begin
+The [!ac](MOOSE) Misc module is designed only to provide code modules that support the solution of physics models that are primarily defined by other MOOSE modules and MOOSE-based applications.
+
+The {{module}} module provides several specializations of MOOSE classes that are used to support calculations performed in a variety of physics calculations. These include `Kernel`, `AuxKernel`, `Material` and `Postprocessor` classes that computing terms in diffusion equations, compute density, internal volume, and perform other similar functions.
```
Comment (cticenhour) [inline modules/misc/doc/content/modules/misc/sqa/misc_srs.md:7]:
```suggestion
The Misc module provides several specializations of MOOSE classes that are used to support calculations performed in a variety of physics calculations. These include `Kernel`, `AuxKernel`, `Material` and `Postprocessor` classes that computing terms in diffusion equations, compute density, internal volume, and perform other similar functions.
```

File: modules/misc/doc/content/modules/misc/sqa/misc_sdd.md [hunk 2]
```diff
@@ -1 +1,26 @@
 !template load file=sqa/module_sdd.md.template category=misc module=Misc
+!template! item key=introduction
```
Comment (cticenhour) [inline modules/misc/doc/content/modules/misc/sqa/misc_sdd.md:2]:
```suggestion

!template! item key=introduction
```

This intro isn't being registered in the rendered site. I think we need to add a space between the `template load` and `template item` commands.

File: modules/misc/doc/content/modules/misc/sqa/misc_sdd.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+ The [!ac](MOOSE) {{module}} module provides a set of models that are used to support a variety of simulations performed using other MOOSE modules or MOOSE-based applications. This module relies on MOOSE for solving the systems of equations that are primarily defined by code outside this module, and simply provides commonly-used code modules that can support those calculations. This document describes the system design of the {{module}} module.
```
Comment (cticenhour) [inline modules/misc/doc/content/modules/misc/sqa/misc_sdd.md:3]:
```suggestion
 The [!ac](MOOSE) {{module}} module provides a set of objects that are used to support a variety of simulations performed using other MOOSE modules or MOOSE-based applications. This module relies on MOOSE for solving the systems of equations that are primarily defined by code outside this module, and simply provides commonly-used code objects that can support those calculations. This document describes the system design of the {{module}} module.
```

File: modules/misc/doc/content/modules/misc/sqa/misc_sdd.md [hunk 4]
```diff
@@ -1 +1,26 @@
...
+
+The {{module}} module provides several specializations of MOOSE classes that are used to support calculations performed in a variety of physics calculations. These include `Kernel`, `AuxKernel`, `Material` and `Postprocessor` classes that computing terms in diffusion equations, compute density, internal volume, and perform other similar functions.
+!template-end!
+
+!template! item key=system-structure
+The [!ac](MOOSE) {{module}} module relies on the MOOSE framework to provide the core functionality of solving multiphysics problems using the finite element method. The structure of the {{module}} module is based on defining C++ classes that derive from classes in the MOOSE framework to provide functionality for solution of a variety of physics problems. By using the interfaces defined in MOOSE base classes for these classes, this module is able to rely on MOOSE to execute these models at the appropriate times during the simulation and use their results in the desired ways.
```
Comment (cticenhour) [inline modules/misc/doc/content/modules/misc/sqa/misc_sdd.md:25]:
```suggestion
The [!ac](MOOSE) {{module}} module relies on the MOOSE framework to provide the core functionality of solving multiphysics problems using the finite element method. The structure of the {{module}} module is based on defining C++ classes that derive from base classes in the MOOSE framework to provide functionality for solution of a variety of physics problems. By using the interfaces defined in MOOSE for these classes, this module is able to rely on MOOSE to execute these models at the appropriate times during the simulation and use their results in the desired ways.
```

File: modules/phase_field/doc/content/modules/phase_field/sqa/phase_field_sdd.md [hunk 5]
```diff
@@ -1 +1,28 @@
...
+The [!ac](MOOSE) {{module}} module inherits the [software dependencies of the MOOSE framework](framework_sdd.md#dependencies-and-limitations), with no additional dependencies.
+!template-end!
+
+!template! item key=design-stakeholders
+!include framework_sdd.md start=design-stakeholders-begin end=design-stakeholders-finish
+!template-end!
+
+!template! item key=system-design
+TODO: Put phase_field-specific text here
+The [!ac](MOOSE) {{module}} module relies on MOOSE to solve the governing equations for teh phase field method, which can include contributions from the thermodynamic potentials provided by this module, elastic energy contributions from mechanical deformation, and temperature fields solved using teh heat conduction module. The design of MOOSE is based on the concept of modular code objects that define all of the aspects of the physics model. This module follows this design, providing code objects that define specific aspects of the solutions for its physics that derive from the base classes defined by the MOOSE framework and the modules that it depends on.
```
Comment (cticenhour) [inline modules/phase_field/doc/content/modules/phase_field/sqa/phase_field_sdd.md:21]:
```suggestion
The [!ac](MOOSE) {{module}} module relies on MOOSE to solve the governing equations for the phase field method, which can include contributions from the thermodynamic potentials provided by this module, elastic energy contributions from mechanical deformation, and temperature fields solved using the heat conduction module. The design of MOOSE is based on the concept of modular code objects that define all of the aspects of the physics model. This module follows this design, providing code objects that define specific aspects of the solutions for its physics that derive from the base classes defined by the MOOSE framework and the modules that it depends on.
```

File: modules/phase_field/doc/content/modules/phase_field/sqa/phase_field_sdd.md [hunk 6]
```diff
@@ -1 +1,28 @@
...
+
+The {{module}} module provides specialized `Kernel` classes that compute the contributions from the terms in the partial differential equations governing the evolution of phase fields. In addition, it provides `IC`, `BC`, `Action`, `Marker`, `MeshGenerator`, `Function`, `AuxKernel`, `Postprocessor` and `VectorPostprocessor` classes to facilitate various aspects of these simulations.
+!template-end!
+
+!template! item key=system-structure
+The [!ac](MOOSE) {{module}} module relies on the MOOSE framework to provide the core functionality of solving multiphysics problems using the finite element method. The structure of the {{module}} module is based on defining C++ classes that derive from classes in the MOOSE framework to provide functionality for solution of phase field problems. By using the interfaces defined in MOOSE base classes for these classes, this module is able to rely on MOOSE to execute these models at the appropriate times during the simulation and use their results in the desired ways.
```
Comment (cticenhour) [inline modules/phase_field/doc/content/modules/phase_field/sqa/phase_field_sdd.md:27]:
```suggestion
The [!ac](MOOSE) {{module}} module relies on the MOOSE framework to provide the core functionality of solving multiphysics problems using the finite element method. The structure of the {{module}} module is based on defining C++ classes that derive from base classes in the MOOSE framework to provide functionality for the solution of phase field problems. By using the interfaces defined in MOOSE for these classes, this module is able to rely on MOOSE to execute these models at the appropriate times during the simulation and use their results in the desired ways.
```

**Review by cticenhour — CHANGES_REQUESTED** [review]
A few adjustments, and then I think we're ready to go here. Thanks all!

**Comment by cticenhour** [comment]
@bwspenc @dschwen Checking in on the status of this one - anything else you wanted to add that we're waiting on?

File: modules/phase_field/doc/content/modules/phase_field/sqa/phase_field_sdd.md (hunk 5 above)
Comment (bwspenc) [reply modules/phase_field/doc/content/modules/phase_field/sqa/phase_field_sdd.md:21]: Not sure what is up with teh keyboard that @dschwen uses 😄

**Comment by bwspenc** [comment]
@cticenhour Sorry about letting this sit for a while. I agreed with your suggestions, and accepted them as a batch in a single commit.

File: modules/misc/doc/content/modules/misc/sqa/misc_sdd.md [hunk 7]
```diff
@@ -1 +1,27 @@
 !template load file=sqa/module_sdd.md.template category=misc module=Misc
+
+!template! item key=introduction
+ The [!ac](MOOSE) {{module}} module provides a set of objects that are used to support a variety of simulations performed using other MOOSE modules or MOOSE-based applications. This module relies on MOOSE for solving the systems of equations that are primarily defined by code outside this module, and simply provides commonly-used code objects that can support those calculations. This document describes the system design of the {{module}} module.
```
Comment (cticenhour) [inline modules/misc/doc/content/modules/misc/sqa/misc_sdd.md:4]:
```suggestion
The [!ac](MOOSE) {{module}} module provides a set of objects that are used to support a variety of simulations performed using other MOOSE modules or MOOSE-based applications. This module relies on MOOSE for solving the systems of equations that are primarily defined by code outside this module, and simply provides commonly-used code objects that can support those calculations. This document describes the system design of the {{module}} module.
```

This space is the bane of CIVET right now...will accept this myself as a new review commit, but I am noting it in the GitHub UI.

**Review by cticenhour — CHANGES_REQUESTED** [review]

**Review by cticenhour — APPROVED** [review]

**Pull request merged**